README.md

```diff
@@ -1 +1,86 @@
-# Math-Magicians
\ No newline at end of file
+# Math Magicians
+
+_Math magicians_ is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+- React
+- [Linters](https://github.com/microverseinc/linters-config/tree/master/react-redux)
+
+## Live Demo Link
+
+[Not Available!]()
+
+## Getting Started
+
```

Comment:
[OPTIONAL] You should include instructions like `git clone ...`. You can check out [this repo](https://github.com/fobadara/math-magicians)

Reply:
Noted!😃